### DISCUSSION TITLE
New MS Teams Icon can't be pinned to taskbar notification area?

### DISCUSSION THREAD

**Discussion by tansbil** [valinet/ExplorerPatcher#2514] [Q&A]
Is it just me or is it not possible to pin the icon for the new MS Teams UI to the taskbar notification area?  I like being able to see what my status is, e.g. to know whether I have unread texts without having to open the Teams app.

**Comment by pyrates999** [comment]
what happens when you try to pin it?  Do you right click on it while it is running and select pin to taskbar?
